### ISSUE TITLE
[Comments] 170417

### ISSUE THREAD

**Issue by bit101** [bit101/lab#107]
Comments for https://bit101.github.io/lab/dailies/170417.html

**Comment by phivk** [comment]
hi @bit101! 

I wanted to say thank you for the nice work you are putting out there. I've been enjoying quicksettings on several projects, e.g. https://unfoldcollective.github.io/bloom/.

Browsing your lab I came across this daily and thought it would fit nicely as the background for a festival site I was building at the time. We've now launched it and I'd love to share it with you:

https://sensorium.is/

![screen shot 2018-03-07 at 23 30 45](https://user-images.githubusercontent.com/902958/37122134-9deeda78-225f-11e8-9d0a-3ca2d19a1774.png)

As you can see, we're inviting visitors to play with the parameters, to which I've made some tweaks.
You can find the sketch as part of the project here:
https://github.com/unfoldcollective/2018.sensorium.is/blob/master/user/themes/imaginarious/js/170417-sensorium.js

Many thanks for sharing your inspiration and keep up the great work!

**Comment by keith-notarize** [comment]
Neat. It would be nice to have a mention on the site.

**Comment by phivk** [comment]
I agree. I plan to publish an announcement of the festival's theme which
ties in with the background design. I think that will be the best place to
mention your work and expand a little on how we integrated it.

I will let you know when that is live.

Neat. It would be nice to have a mention on the site.

**Comment by phivk** [comment]
hi, sorry for the long wait!
I've finally gotten round to writing that piece on the festival theme and have added a mention to you on the [about page](https://sensorium.is/about) and in the footer.

I'm linking to http://www.bit-101.com/ and https://bit101.github.io/lab/thumbs.html

Are you cool with those links, or would you rather have us link to somewhere else?
